blocks/components/editable/index.js

```diff
@@ -152,46 +158,23 @@ export default class Editable extends wp.element.Component {
 			return;
 		}
 
-		// Getting the content before and after the cursor
-		const childNodes = Array.from( this.editor.getBody().childNodes );
-		let selectedChild = this.editor.selection.getStart();
-		while ( childNodes.indexOf( selectedChild ) === -1 && selectedChild.parentNode ) {
-			selectedChild = selectedChild.parentNode;
-		}
-		const splitIndex = childNodes.indexOf( selectedChild );
-		if ( splitIndex === -1 ) {
+		if ( ! this.content.length > 1 ) {
```

Comment:
Heh, I'm surprised this isn't captured by [ESLint's `no-unsafe-negation`](http://eslint.org/docs/rules/no-unsafe-negation), as there's a very subtle bug here:

```
( ! 5 > 1 )
// false
( ! 0 > 1 )
// false
```

The issue is that the boolean coercion occurs before the `>` comparison, and both `true > 1` and `false > 1` evaluate to `false`.

In this case, I'm guessing your intention is for the condition to match when `length` is zero, in which case it should be enough to change this to just `if ( ! this.content.length ) {`